src/pages/users/UsersPage.tsx

```diff
@@ -60,6 +64,21 @@ const UsersPage: FunctionComponent = () => {
     const { snackError } = useSnackMessage();
     const gridRef = useRef<GridTableRef<UserInfos>>(null);
     const gridContext = gridRef.current?.context;
+    const [profileNameOptions, setprofileNameOptions] = useState<string[]>([]);
+
+    useEffect(() => {
+        UserAdminSrv.fetchProfiles().then((allProfiles: UserProfile[]) => {
+            let profiles: string[] = [
+                intl.formatMessage({ id: 'users.table.profile.none' }),
+            ];
+            if (allProfiles?.length) {
```

Comment:
explicit !== 0, otherwise it's confusing

Reply:
if allProfiles is undefined, we would have :
undefined !== 0 -> true